pkg/kubectlcmd/client.go

```diff
@@ -102,6 +102,103 @@ func (c *Client) Apply(dryRun, verbose bool, kubeconfig, context, namespace stri
 	return stdout.String(), csError, nil
 }
 
+// GetAll runs the kubectl get all with the provided arguments
```

Comment:
Please add a bit more description for the params and return values.

Reply:
Added.